zeebe/atomix/cluster/pom.xml

```diff
@@ -272,6 +272,12 @@
       <artifactId>netty-resolver</artifactId>
     </dependency>
 
+    <dependency>
+      <groupId>org.springframework</groupId>
```

Comment:
:x:  why did you need to add this? I don't think it's correct

Reply:
Since we're using `DataSize` in `MessagingConfig` it became a dependency caught by the build validations.